src/pages/index.js

```diff
@@ -1,101 +1,245 @@
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import "./index.css";
-import { initialCards, config } from "../utils/constants.js";
+import { config } from "../utils/constants.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupConfirmDelete from "../components/PopupConfirmDelete.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 
-// ELEMENTS////
+/* -------------------------------------------------------------------------- */
+/*                                  Elements                                  */
+/* -------------------------------------------------------------------------- */
 
 const profileEditBtn = document.querySelector("#profile__edit-button");
-const profileEditModal = document.querySelector("#profile-edit-modal");
-
-const profileTitle = document.querySelector(".profile__title");
-const profileDescrption = document.querySelector(".profile__description");
 const modalInputTitle = document.querySelector("#modal__input-title");
 const profileInputDescrption = document.querySelector(
   "#profile__description-input"
 );
-
-const profileAddModal = document.querySelector("#profile-add-modal");
+const avatarEditBtn = document.querySelector("#profile__avatar");
+/* -------------------------------------------------------------------------- */
+/*                                  ADD NEW CARD                                  */
+/* -------------------------------------------------------------------------- */
 const addNewCardButton = document.querySelector(".profile__add-button");
-
-// const cardForm = profileAddModal.querySelector(".modal__form");
-
 const cardForm = document.forms["card_form"];
 const profileForm = document.forms["profile_form"];
-// ADD NEW CARD /////
-
-const addFormValidator = new FormValidator(config, cardForm);
-const editFormValidator = new FormValidator(config, profileForm);
-addFormValidator.enableValidation();
-editFormValidator.enableValidation();
+const avatarForm = document.forms["edit-avatar-form"];
+
+/* -------------------------------------------------------------------------- */
+/*                                  Api                                   */
+/* -------------------------------------------------------------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "eac0c1a2-60c5-44c4-979d-14695d75b6b5",
+    "Content-Type": "application/json",
+  },
+});
 
 const section = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: renderCard,
   },
   ".cards__list"
 );
 section.renderItems();
```

Comment:
`section.renderItems();
`

It's not needed here because you do that in line 54 after setting the array